### PR TITLE
numpy instance invert

### PR DESCRIPTION
**Pull request by KartikeyDubeyKD**
close https://github.com/unifyai/ivy/issues/14752

### CODE REVIEW

**Comment by KartikeyDubeyKD** [comment]
close #14752

**Comment by KartikeyDubeyKD** [comment]
Hi @YushaArif99, I've just submitted a PR that adds a new function to the project. You can find it at #14752. Could you please take a look and let me know your thoughts on the implementation? Thanks in advance!

File: ivy/functional/frontends/numpy/__init__.py [hunk 1]
```diff
@@ -703,3 +703,4 @@ def promote_types_of_numpy_inputs(
 rint = ufunc("_rint")
 nextafter = ufunc("_nextafter")
 conjugate = ufunc("_conjugate")
+invert = ufunc("_invert")
```
Comment (YushaArif99) [inline ivy/functional/frontends/numpy/__init__.py:706]: this line seems to be causing an error: 

```
E  AttributeError: partially initialized module 'ivy.functional.frontends.numpy' has no attribute '_invert' (most likely due to a circular import)
```

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @KartikeyDubeyKD 

The tests for `test_numpy_instance_invert__` seem to be failing in the CI ([here](https://github.com/unifyai/ivy/actions/runs/4845549426/jobs/8634529700?pr=14755)) due to the abovementioned problem. Can you kindly look into this and re-request a review once all relevant tests are passing? Thanks 🙂

**Comment by KartikeyDubeyKD** [comment]
apologies ma'am can you please take a look and tell me what I'm doing wrong?

**Comment by KartikeyDubeyKD** [comment]
@YushaArif99 ma'am can you please what I'm doing wrong why are the rest 10 checks are not passing

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @KartikeyDubeyKD 

I'm not sure exactly where the error is but the tests are still failing with a different error this time. 
```
E   RuntimeError: linalg.inv: The input tensor A must have at least 2 dimensions.
```

I think your inputs to the `inv()` are incorrect. Its best to view the official document of NumPy to better understand the input requirements of the function and appropriately generate those inputs using hypothesis. 

Once the CI tests start passing, you can re-request a review. Thanks! 🙂

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 2]
```diff
@@ -2625,6 +2625,40 @@ def test_numpy_instance_abs__(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="__invert__",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        min_num_dims=0,
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:2634]: you've set `min_num_dims=0` here hence scalar values will also be generated. Whereas, the error indicates that the minimum no of dimensions should be 2.

Comment (KartikeyDubeyKD) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:2634]: got it

**Comment by YushaArif99** [comment]
Hi @KartikeyDubeyKD 

Are you still working on this PR? If not, we can then close this PR 🙂

**Comment by KartikeyDubeyKD** [comment]
Please do not close this PR, I'm working on it. Something important came up that's why I was on hold.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**